src/test/java/org/jabref/gui/edit/ReplaceStringViewTest.java

```diff
@@ -0,0 +1,81 @@
+package org.jabref.gui.edit;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+
+import javafx.scene.control.*;
+import javafx.stage.Stage;
+
+import org.jabref.gui.LibraryTab;
+import org.jabref.model.database.BibDatabase;
+import org.jabref.model.database.BibDatabaseContext;
+import org.jabref.model.entry.BibEntry;
+import org.jabref.model.entry.field.StandardField;
+import org.jabref.model.entry.types.StandardEntryType;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import org.junit.jupiter.api.Test;
+
+import org.testfx.framework.junit5.ApplicationTest;
+
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+public class ReplaceStringViewTest extends ApplicationTest {
+
+    private ReplaceStringView replaceStringView;
+    private LibraryTab libraryTab = mock(LibraryTab.class);
+    private BibDatabaseContext databaseContext = mock(BibDatabaseContext.class);
+    private DialogPane dialogPane;
+    private BibEntry entry;
+
+    @Override
+    public void start (Stage stage) throws Exception {
+        // Globals.prefs = JabRefPreferences.getInstance();
+        // Globals.startBackgroundTasks();
```

Comment:
These are the lines of code that helped my test to run, because it needs Globals.prefs and Globals.fileUpdateMonitor to not be null.

Reply:
Just an idea, can you try if Globals.prefs = mock(JabRefPreferences.class) works?
for Globals.fileUpdateMonitor you can try to use the DummyFileUpdateMonitor

Reply:
@Siedlerchr  Yes it would work if we mock it up but I think the point is that we should not assign new value to Globals.prefs like this "Globas.prefs = ...." in the test. Because it's not like creating an instance of the object Globals and mock its prefs, but now we are directly setting the value of this global variable which is not ideal I think.

Reply:
@JabRef/developers  anyone an idea how to either extract the globals or how to test it?